ime/app/src/main/java/com/anysoftkeyboard/AnySoftKeyboard.java

```diff
@@ -444,6 +444,9 @@ private void onFunctionKey(final int primaryCode, final Key key, final boolean f
                     sendDownUpKeyEvents(keyEventKeyCode);
                 }
                 break;
+            case KeyCodes.SPACE:
```

Comment:
I don't think this code can ever execute. The method `onFunctionKey` is only called if the key-code is a negative value (which SPACE is not).

Reply:
Oh, we don't need that. I was just following other examples in the code and making a copy for "space". I've removed it.